games/xworld/maps/xworld_env.py

```diff
@@ -54,13 +54,16 @@ class XWorldEnv(object):
 
     curriculum_check_period = 100
 
-    def __init__(self, item_path, max_height=7, max_width=7):
+    def __init__(self, item_path, max_height=7, max_width=7, class_per_session=-1):
```

Comment:
don't put this in the base class; do this in your map file instead. See xworld3d/maps/XWorld3DNav.py

Reply:
Done.